xc7/archs/artix7/devices/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_xc7_device_define(
   ARCH artix7
-  DEVICES xc7a50t-basys3 xc7a50t-arty
-  PARTS xc7a35tcpg236-1 xc7a35tcsg324-1
+  DEVICES xc7a50t-basys3 xc7a50t-arty xc7a50t-arty-swbut
```

Comment:
I would also rename `xc7a50t-arty` to `xc7a50t-arty-uart` so to make clear that one is uart and the other is swbut